include/openssl/core.h

```diff
@@ -143,6 +143,10 @@ struct ossl_param_st {
  */
 # define OSSL_PARAM_OCTET_PTR            7
 
+/* Typedef for thread stop handling callback */
```

Comment:
Considering the rest of this header file, that comment seems a bit...  minimal

Reply:
Fixup pushed expanding out that comment.